signcrypt_seal.go

```diff
@@ -69,6 +69,7 @@ func (sss *signcryptSealStream) signcryptBytes(b []byte) error {
 
 	nonce := nonceForChunkSigncryption(sss.numBlocks)
 
+	// TODO: Encode payload chunk.
```

Comment:
Ah that refers to the packet as a whole, which is getting encoded as a 1-element list on line 99 below. This was an open question when the only element in the packet was this secretbox-of-a-signature, since we could've decided to put that directly on the wire as a bin object. Though now that we've settled on having a final packet flag, there's no longer any question that the packet will be a list.